react-toggle-switch-component/src/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isClicked: !this.state.isClicked });
+  }
+
+  render() {
+    return (
+      <div className='container'>
+      <div>
+        <button className= {this.state.isClicked ? 'toggle-container on' : 'toggle-container'} onClick={this.handleClick}>
+          <button className={this.state.isClicked ? 'toggle on' : 'toggle'}></button>
+        </button>
+      </div>
+        <h2>{this.state.isClicked ? 'ON' : 'OFF'}</h2>
+      </div>
+    );
+  }
+}
```

Comment:
Your button component definition is in the wrong file:

---

![image](https://user-images.githubusercontent.com/16143771/169583892-a2122d4d-799e-4b43-a104-25a9d206894b.png)